Code/Framework/AzCore/AzCore/Statistics/StatisticalProfilerProxy.h

```diff
@@ -135,30 +131,55 @@ namespace AZ::Statistics
         StatisticalProfilerProxy(StatisticalProfilerProxy&&) = delete;
         StatisticalProfilerProxy& operator=(StatisticalProfilerProxy&&) = delete;
 
+        void RegisterProfilerId(StatisticalProfilerId id)
+        {
+            m_profilers.try_emplace(id, AZStd::move(ProfilerInfo()));
```

Comment:
Calling `move` on a temporary isn't needed.
It is already a rvalue.

```suggestion
            m_profilers.try_emplace(id, ProfilerInfo());
```

Reply:
Yea, realized that right after posting. Already fixed locally.